### PR TITLE
chore, refactor(egen): format and refactor for get_started_with_model_monitoring_custom_tf_serving notebook

### PR DESCRIPTION
**Pull request by k-root**
Following changes are made as part of the PR:
- REGION updated to LOCATION
- Removes boilerplate
- Formats according to template

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 1]
```diff
@@ -75,21 +80,21 @@
...
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Model Monitoring`\n",
-        "- `Vertex AI Prediction`\n",
-        "- `Vertex AI Model` resource\n",
-        "- `Vertex AI Endpoint` resource\n",
+        "- Vertex AI Model Monitoring\n",
+        "- Vertex AI Prediction\n",
+        "- Vertex AI Model resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:89]: lower case: "Vertex AI model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 2]
```diff
@@ -75,21 +80,21 @@
...
+        "- Vertex AI Endpoint resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:90]: lower case: "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 3]
```diff
@@ -75,21 +80,21 @@
...
         "\n",
         "The steps performed include:\n",
         "\n",
         "- Download a pre-trained custom tabular model.\n",
-        "- Upload the pre-trained model as a `Model` resource.\n",
-        "- Deploying the `Model` resource to an `Endpoint` resource with `TensorFlow Serving` serving binary.\n",
-        "- Configure the `Endpoint` resource for model monitoring.\n",
+        "- Upload the pre-trained model as a Model resource.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:95]: lower case: "model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 4]
```diff
@@ -75,21 +80,21 @@
...
+        "- Deploying the Model resource to an Endpoint resource with \"TensorFlow Serving\" serving binary.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:96]: lower case "endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 5]
```diff
@@ -754,12 +705,12 @@
         "\n",
         "The following are the basic steps to enable model monitoring:\n",
         "\n",
-        "1. Deploy a `Vertex AI` AutoML or custom tabular model to an `Vertex AI Endpoint`.\n",
+        "1. Deploy a Vertex AI AutoML or custom tabular model to an Vertex AI Endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:708]: lower case: "Vertex AI endpoint"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 6]
```diff
@@ -754,12 +705,12 @@
...
         "2. Configure a model monitoring specification.\n",
-        "3. Upload the model monitoring specification to the `Vertex AI Endpoint`.\n",
+        "3. Upload the model monitoring specification to the Vertex AI Endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:710]: lower case: "Vertex AI endpoint"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 7]
```diff
@@ -754,12 +705,12 @@
...
         "4. Upload or automatic generation of the `input schema` for parsing.\n",
         "5. For feature skew detection, upload the training data for automatic generation of the feature distribution.\n",
-        "6. For feature attributions, upload corresponding `Vertex AI Explainability` specification.\n",
+        "6. For feature attributions, upload corresponding Vertex AI Explainability specification.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:713]: incorrect product name. Should be "Vertex Explainable AI"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 8]
```diff
@@ -769,7 +720,7 @@
         "\n",
         "For skew detection, the monitoring service requires a baseline for the statistical distribution of values in the training data. For AutoML tabular models this is automatically derived. For custom tabular models, you upload the training data to the service, and have the service automatically derive the distribution.\n",
         "\n",
-        "For feature attribution skew and drift detection, requires enabling your deployed model for `Vertex AI Explainability` for custom tabular models. For AutoML models, `Vertex AI Explainability` is automatically enabled.\n",
+        "For feature attribution skew and drift detection, requires enabling your deployed model for Vertex AI Explainability for custom tabular models. For AutoML models, Vertex AI Explainability is automatically enabled.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:723]: incorrect product name. Should be "Vertex Explainable AI"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 9]
```diff
@@ -805,13 +756,13 @@
         "id": "9bf06cd476e9"
       },
       "source": [
-        "### Upload the model artifacts as a `Vertex AI Model` resource\n",
+        "### Upload the model artifacts as a Vertex AI Model resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:759]: lower case: "Vertex AI model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 10]
```diff
@@ -805,13 +756,13 @@
...
         "\n",
-        "First, you upload the pre-trained custom tabular model artifacts as a `Vertex AI Model` resource using the `upload()` method, with the following parameters:\n",
+        "First, you upload the pre-trained custom tabular model artifacts as a Vertex AI Model resource using the `upload()` method, with the following parameters:\n",
         "\n",
-        "- `display_name`: The human readable name for the `Model` resource.\n",
+        "- `display_name`: The human readable name for the Model resource.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:763]: lower case "model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 11]
```diff
@@ -805,13 +756,13 @@
         "id": "9bf06cd476e9"
       },
       "source": [
-        "### Upload the model artifacts as a `Vertex AI Model` resource\n",
+        "### Upload the model artifacts as a Vertex AI Model resource\n",
         "\n",
-        "First, you upload the pre-trained custom tabular model artifacts as a `Vertex AI Model` resource using the `upload()` method, with the following parameters:\n",
+        "First, you upload the pre-trained custom tabular model artifacts as a Vertex AI Model resource using the `upload()` method, with the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:761]: lower case: "Vertex AI model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 12]
```diff
@@ -828,7 +779,7 @@
         "\n",
         "*Note:* You drop the ending number subfolder (e.g., /1) from the model path to upload. The Vertex service will upload the parent folder above the subfolder with the model artifacts -- which is what TensorFlow Serving binary expects.\n",
         "\n",
-        "*Note:* When you upload the model artifacts to a `Vertex AI Model` resource, you specify the corresponding deployment container image."
+        "*Note:* When you upload the model artifacts to a Vertex AI Model resource, you specify the corresponding deployment container image."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:782]: lower case: "Vertex AI model resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 13]
```diff
@@ -868,9 +819,9 @@
         "id": "1069c6f0eba8"
       },
       "source": [
-        "### Deploy the `Vertex AI Model` resource to a `Vertex AI Endpoint` resource\n",
+        "### Deploy the Vertex AI Model resource to a Vertex AI Endpoint resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:822]: lower case: "Vertex AI model resource" and "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 14]
```diff
@@ -868,9 +819,9 @@
...
         "\n",
-        "Next, you deploy your `Vertex AI Model` resource to a `Vertex AI Endpoint` resource using the `deploy()` method, with the following parameters:\n",
+        "Next, you deploy your Vertex AI Model resource to a Vertex AI Endpoint resource using the `deploy()` method, with the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:824]: lower case: "Vertex AI model resource" and "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 15]
```diff
@@ -1197,7 +1148,7 @@
         "- `display_name`: The human readable name for the monitoring job.\n",
         "- `project`: The project ID.\n",
         "- `region`: The region.\n",
-        "- `endpoint`: The fully qualified resource name of the `Vertex AI Endpoint` to enable monitoring.\n",
+        "- `endpoint`: The fully qualified resource name of the Vertex AI Endpoint to enable monitoring.\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:1151]: lower case: "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 16]
```diff
@@ -1514,7 +1465,7 @@
       "source": [
         "### Make the prediction requests\n",
         "\n",
-        "Next, you send the the 1000 prediction requests to your `Vertex AI Endpoint` resource using the `predict()` method."
+        "Next, you send the the 1000 prediction requests to your Vertex AI Endpoint resource using the `predict()` method."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:1468]: lower case: "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 17]
```diff
@@ -1627,9 +1578,9 @@
         "id": "c6280efab664"
       },
       "source": [
-        "#### Undeploy and delete the `Vertex AI Endpoint` resource\n",
+        "#### Undeploy and delete the Vertex AI Endpoint resource\n",
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:1581]: lower case: "Vertex AI endpoint resource"

File: notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb [hunk 18]
```diff
@@ -1627,9 +1578,9 @@
...
         "\n",
-        "Your `Vertex AI Endpoint` resource can be deleted using the `delete()` method. Prior to deleting, any model deployed to your `Vertex AI Endpoint` resource, must first be undeployed."
+        "Your Vertex AI Endpoint resource can be deleted using the `delete()` method. Prior to deleting, any model deployed to your Vertex AI Endpoint resource, must first be undeployed."
```
Comment (kittyabs) [inline notebooks/official/model_monitoring/get_started_with_model_monitoring_custom_tf_serving.ipynb:1583]: lower case: "Vertex AI endpoint resource" (2x's)

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Comment by k-root** [comment]
Addressed all comments. Please review

**Review by gericdong — APPROVED** [review]

**Pull request merged**